debian/rules

```diff
@@ -34,6 +34,9 @@ override_dh_auto_test:
 
 override_dh_auto_install:
 	DESTDIR=$(CURDIR)/debian/tmp ninja -C build-deb install
+	# Debian kmod has a conf file called mlx4.conf that unnecessarily
+	# tries to autoload mlx4_en, call our conf file rdma-mlx4.conf instead
+	mv $(CURDIR)/debian/tmp/etc/modprobe.d/mlx4.conf $(CURDIR)/debian/tmp/etc/modprobe.d/rdma-mlx4.conf
```

Comment:
mlx4.conf is only part of Ubuntu's kmod package. See https://launchpad.net/ubuntu/+source/kmod/24-1ubuntu1 and https://bugs.launchpad.net/maas/+bug/1115710

I am against renaming the file. kmod should stop providing that file and rely on rdma-core.